.github/workflows/destroy.yml

```
@@ -8,14 +8,14 @@ on:
         required: true
 
 jobs:
-  terraform-destroy-resources:
-    name: Destroy Application Resources
+  terraform-destroy-all:
+    name: Destroy Application and Backend Resources
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)